### ISSUE TITLE
document keydown event no longer fires for function keys or escape

### ISSUE THREAD

**Issue by wravery** [MicrosoftEdge/WebViewFeedback#63]
I think the only thing that changed was I updated Edge to build `79.0.308.1` on the Dev Channel. I'm still on the current WebView2 NuGet version `0.8.270`.

I'm currently handling some accelerators on the page by posting a web message back to the app. I also noticed that none of the `Control + *` accelerators from Edge are working anymore in the webpage. I was able to reproduce this behavior in both a test harness and in an integrated build.

Simplified version of the event handler I'm using:

```javascript
document.addEventListener("keydown", ev => {
  // Used to be called for "F6" and "Escape"...
  const key = getAccelerator(ev.key);
  if (key) {
    // post the web message
    window.chrome.webview.postMessage({ ... });
  }
});
```

**Comment by david-risney** [comment]
Thanks. Let me check if this is a regression we've just recently fixed or a new bug.

**Comment by david-risney** [comment]
This should be fixed in 79.0.313.0
